common/src/test/scala/org/apache/celeborn/common/util/PbSerDeUtilsTest.scala

```diff
@@ -101,6 +101,7 @@ class PbSerDeUtilsTest extends CelebornFunSuite {
       1005,
       diskInfos,
       userResourceConsumption)
+  workerInfo1.networkLocation_$eq("/1")
```

Comment:
Any assert for network location?

Reply:
I don't think we need an assert for network location, since it serializes and deserializes, and then asserts both are same here: https://github.com/apache/celeborn/blob/main/common/src/test/scala/org/apache/celeborn/common/util/PbSerDeUtilsTest.scala#L253-L260